### PR TITLE
Add "bird's eye" picture to ARCHITECTURE.md

### PR DESCRIPTION
**Pull request by 0sewa0**
## Description

It just adds a little picture do go along with the "Birds's Eye View" section
(Accidentally force-pushed over this change when adding the `ARCHITECTURE.md`.) 

## How can this be tested?

look at it 😅  (the "rich diff" and not the "raw" one)

## Checklist

- ~[ ] Unit tests have been updated/added~
- [x] PR is labeled accordingly with a single label
- [x] I have read and understood the [contribution guidelines](https://github.com/Dynatrace/dynatrace-operator/blob/main/CONTRIBUTING.md)

### CODE REVIEW

File: ARCHITECTURE.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ This document describes the high-level architecture of `Dynatrace Operator`.
 If you want to familiarize yourself with the code base, you are just in the right place!
 
 ## Bird's Eye View
-<picture-here>
+![image](https://github.com/Dynatrace/dynatrace-operator/assets/31651557/ebc8973d-2c2c-4f9a-af32-0503b65e47af)
 
 On a very high level, what the operator does is for a given `CustomResource`(CR) provided by the user, the `Operator` will deploy _one or several_ Dynatrace components into the Kubernetes Environment.
```
Comment (waodim) [inline ARCHITECTURE.md:8]:
```suggestion
On a very high level, what the `Operator` does is for a given `CustomResource`(CR) provided by the user, the `Operator` will deploy _one or several_ Dynatrace components into the Kubernetes Environment.
```

File: ARCHITECTURE.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ This document describes the high-level architecture of `Dynatrace Operator`.
 If you want to familiarize yourself with the code base, you are just in the right place!
 
 ## Bird's Eye View
-<picture-here>
+![image](https://github.com/Dynatrace/dynatrace-operator/assets/31651557/ebc8973d-2c2c-4f9a-af32-0503b65e47af)
```
Comment (andriisoldatenko) [inline ARCHITECTURE.md:6]:
```suggestion
```mermaid
graph LR
    A[fa:fa-user User] -->|Creates| B(fa:fa-file CR)
    subgraph kubernetes
    B --> |triggers| C(fa:fa-wrench Operator)
    C -->|deploys| D[Dynatrace-Compontent-1]:::dk1
    C -->|deploys| E[Dynatrace-Compontent-2]:::dk2
    C -->|deploys| F[Dynatrace-Compontent-3]:::dk3
    end

    classDef dk1 stroke:#f00
    classDef dk2 stroke:#0f0
    classDef dk3 stroke:#00f
```
```

**Review by andriisoldatenko — DISMISSED** [review]
🎉

**Comment by albertogdd** [comment]
Birds eye view diagram has a typo > "Dynatrace-Compontnent-X"

File: ARCHITECTURE.md [hunk 3]
```diff
@@ -3,7 +3,20 @@ This document describes the high-level architecture of `Dynatrace Operator`.
...
 ## Bird's Eye View
-<picture-here>
+```mermaid
+graph LR
+    A[fa:fa-user User] -->|Creates| B(fa:fa-file CR)
+    subgraph kubernetes
+    B --> |triggers| C(fa:fa-wrench Operator)
+    C -->|deploys| D[Dynatrace-Compontent-1]:::dk1
+    C -->|deploys| E[Dynatrace-Compontent-2]:::dk2
+    C -->|deploys| F[Dynatrace-Compontent-3]:::dk3
```
Comment (albertogdd) [inline ARCHITECTURE.md:13]:
```suggestion
    C -->|deploys| D[Dynatrace-Component-1]:::dk1
    C -->|deploys| E[Dynatrace-Component-2]:::dk2
    C -->|deploys| F[Dynatrace-Component-3]:::dk3
```

**Review by albertogdd — DISMISSED** [review]

File: ARCHITECTURE.md [hunk 4]
```diff
@@ -3,7 +3,20 @@ This document describes the high-level architecture of `Dynatrace Operator`.
 If you want to familiarize yourself with the code base, you are just in the right place!
 
 ## Bird's Eye View
-<picture-here>
+```mermaid
+graph LR
+    A[fa:fa-user User] -->|Creates| B(fa:fa-file CR)
```
Comment (chrismuellner) [inline ARCHITECTURE.md:8]:
```suggestion
    A[fa:fa-user User] -->|creates| B(fa:fa-file CR)
```

**Review by chrismuellner — CHANGES_REQUESTED** [review]

**Review by waodim — APPROVED** [review]

**Pull request merged**